tracing-journald/src/lib.rs

```diff
@@ -174,12 +179,34 @@ where
             self.field_prefix.as_ref().map(|x| &x[..]),
         ));
 
-        // What could we possibly do on error?
         #[cfg(unix)]
-        let _ = self.socket.send(&buf);
+        #[allow(dead_code)]
+        let result = self.socket.send(&buf).map_err(|e| e.raw_os_error());
+
+        #[cfg(target_os = "linux")]
```

Comment:
Maybe at this point it's simpler to `cfg` out the entire function in favor of a stub on nonlinux?

Reply:
Indeed.  I wanted to keep the structure of the original code, but it's gotten unnecessarily complex.

If moved the whole send logic into a single `send_payload` function which is stubbed out on non-Linux.